libs/coin-evm/src/logic.ts

```diff
@@ -44,6 +44,12 @@ export const getEstimatedFees = (tx: EvmTransaction): BigNumber => {
   return tx.maxFeePerGas?.multipliedBy(gasLimit) || new BigNumber(0);
 };
 
+/**
+ * Helper to get the currency unit to be used for the fee field
+ */
+export const getDefaultFeeUnit = (currency: CryptoCurrency): Unit =>
+  currency.units.length > 1 ? currency.units[1] : currency.units[0];
```

Comment:
Correct me if i'm wrong, but i understand we suppose there is always either 2 units or 1, shouldn't this be the last element then ?

Reply:
Also it happens there's more than 2 elements. For exemple ethereum has 4.

https://github.com/LedgerHQ/ledger-live/blob/fd35196740b2baa0247be755043225300688024e/libs/ledgerjs/packages/cryptoassets/src/currencies.ts#L51